src/run/runner/wall_time/executor.rs

```diff
@@ -38,15 +44,80 @@ impl Executor for WallTimeExecutor {
             cmd.current_dir(abs_cwd);
         }
 
-        cmd.args(["-c", get_bench_command(config)?.as_str()]);
+        // Create the perf FIFOs before starting the command
+        let mut perf_fifo = PerfFifo::new();
+        let mut ctl_fifo = FifoIpc::create(RUNNER_CTL_FIFO)
+            .unwrap()
```

Comment:
Let's banish `unwrap()` usage altogether in favor of an anyhow error or at the very least an `expect` 😊